sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -106,8 +109,9 @@ def filter_words(unused_x):
   p.visit(SnippetUtils.RenameFiles(renames))
 
   # [START pipelines_constructing_running]
-  p.run()
+  result = p.run()
   # [END pipelines_constructing_running]
+  result
```

Comment:
No, leftover code. Removed it. (Did a a global search to find other left over code but could not.)